Doc/library/decimal.rst

```diff
@@ -1215,6 +1222,13 @@ In addition to the three supplied contexts, new contexts can be created with the
       Returns ``True`` if *x* is infinite; otherwise returns ``False``.
 
 
+   .. method:: is_integer(x)
+
+      Returns ``True`` if *x* is finite and integral; otherwise
```

Comment:
One day it would be nice to fix all these docstrings for consistency (both with one another and with PEP 257). But not today.

Reply:
Indeed, when you hop around the code you notice how different they all are. I tried to go for local consistency rather than global consistency.